main/midi/midiPlayer.c

```diff
@@ -1249,6 +1249,17 @@ void midiPlayerReset(midiPlayer_t* player)
     player->paused = true;
 }
 
+void midiPlayerResetNewSong(midiPlayer_t* player)
+{
+    midiAllSoundOff(player);
+
+    // Set all the relevant bits to 1, meaning not in use
+    player->percSpecialStates = 0b00111111111111111111111111111111; // 0x4fffffff
+
+    player->sampleCount    = 0;
+    player->eventAvailable = false;
+}
+
```

Comment:
This is basically a version of `midiPlayerReset` that does the bare minimum needed to play multiple sound effects in quick succession without causing stuck notes. I was not sure of a function name that would make sense.

Reply:
Thanks for the note here, I'm gonna turn that into a doxy comment :laughing: 

Once the slack notification hits, you should let people know this function exists and when it should be used.